README.adoc

```diff
@@ -16,7 +16,7 @@ Or see the Hugo installation instructions here: https://gohugo.io/getting-starte
 
 On Fedora: `sudo dnf install asciidoctor`
```

Comment:
Just a note, that for me (Fedora) a install of asciidoctor was not enough and I had to run a bundle install.

Reply:
Yes...We should remove it. I'll do